docassemble/MATCSmallClaims/data/questions/small_claims.yml

```diff
@@ -203,40 +210,46 @@ validation code: |
   if not acknowledged_information_use:
     validation_error("You must accept the terms of use to continue.", field="acknowledged_information_use")
 ---
-id: Small_claims
+id: small claims interview info
 continue button field: small_claims_intro
 question: |
   Information for the small claims complaint process
 subquestion: |
-  This interview will help you file a small claims complaint.
+  This interview will help you file a small claims complaint. In order to complete the 
+  interview you will need to know:
   
-  Before you get started, gather:
+  1. The full name, address, and phone number or email address of anyone you intend to sue
+  2. A summary or reason why you are suing them
+  3. The military status of anyone you intend to sue (information about verifying military status will be provided)
   
-  1. Address of anyone you intend to sue.
-  2. Why you are suing them.
-  3. Military statuses of anyone you intend to sue. 
-  
-  When you are finished, you will need to:
+  When you are finished, you can either:
+
+  * File your claim electronically, or
+  * Download and print your claim forms so you can deliver them to the courthouse yourself
   
-  1. Pay the filing fee. 
+  You will also need to pay the filing fee. 
 
-  Amount you are claiming | Filing Fee 
-           ------|----
-    $500 or less | $40
-    $501 - $2,000 | $50
-    $2,001 - $5,000| $100 
-    $5,001 - $7,000| $150
-    More than $7,000 | $150   
+  Amount of your claim | Filing fee 
+  ---------------------|-----------
+  $500 or less         | $40
+  $501–2,000           | $50
+  $2,001–5,000         | $100 
+  $5,001–7,000         | $150
+  More than $7,000     | $150   
   
-  If you cannot afford to pay your filing fee, you can ask the court to waive it using this interview.
+  If you cannot afford to pay the filing fee you can ask the court for a waiver. You can 
+  complete a fee-waiver form during this interview.
   
-  Please note that this interview cannot tell you what you should say to meet the legal basis needed for a judge to rule in your favor. The court can provide information about how to file a claim, but cannot give legal advice.
+  Please note that this interview cannot tell you what you should say to meet the legal 
```

Comment:
Will adding newlines affect how the text displays on the front end, particularly with wrapping and responsiveness on mobile devices? I'm concerned it might lead to layouts that look something like this:

This is text here
that
we added newlines
for
so now it looks a
little weird.

Reply:
content in this area is styled with Markdown so you need an empty line to add a newline.

Reply:
Wouldn't the text auto-wrap when displayed?

Reply:
It ends up treating a single newline the same as no newlines,.and 2 newlines makes a new paragraph

Reply:
In the LIT Lab's code I've noticed that the preference seems to be to use newlines rather than relying on word wrapping, so that's what I did here. Since most people will be seeing it in the same editor (the Playground) hopefully it doesn't result in weird hanging lines.

There are some quirks. I think it usually interprets a newline as a space, and I haven't figured out how they interact with lists, so I leave them out there.